frontend/src/sass/pages/_permit-guidelines.scss

```diff
@@ -1,4 +1,4 @@
-.permit-guidelines {
+//.permit-guidelines {
```

Comment:
Can we clean up / remove any commented code

Reply:
Commented code removed